plugins/mapper-annotated-text/src/main/java/org/elasticsearch/index/mapper/annotatedtext/AnnotatedTextFieldMapper.java

```diff
@@ -75,14 +75,14 @@
 import static org.elasticsearch.index.mapper.TypeParsers.parseTextField;
 
 /** A {@link FieldMapper} for full-text fields with annotation markup e.g.
- * 
```

Comment:
Could you extract the formatting changes to this file into another commit so I don't have to review them? I'd be happy with you just pushing it so long as all it does is remove trailing spaces.

Reply:
Sure, I didn't realize this did a reformatting. Will clean this up.